encrypted-media-respec.html

```diff
@@ -3611,6 +3611,12 @@
                 <li><p>Data, including <a def-id="distinctive-identifiers"></a> and <a def-id="distinctive-permanent-identifiers"></a>, transmitted between the client and server can be viewed and/or modified by other entities. See <a href="#user-tracking">User Tracking</a>.</p></li>
               </ul>
             </li>
+            <li>
```

Comment:
I moved this from line 3877, which I replaced with a more general reference to this section.